content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
+[Architecture](/docs/started/architecture/) pages.
 
 ### How to install Kubeflow?
 
-There are two pathways to get up and running with Kubeflow, you may either:
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+There are two primary ways to install Kubeflow:
 
-1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
-2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
+1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
+1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
 
 <a id="packaged-distributions"></a>
 <a id="install-a-packaged-kubeflow-distribution"></a>
-## Packaged Distributions of Kubeflow 
+
+## Packaged Distributions of Kubeflow
+
+Packaged distributions are maintained by various organizations and typically aim to provide
+a simplified installation and management experience for Kubeflow.
 
 {{% alert title="Note" color="warning" %}}
-Packaged distributions are developed and supported by their respective maintainers, <b>the Kubeflow community does not endorse or certify any specific distribution</b>.
+Packaged distributions are developed and supported by their respective maintainers.
+The Kubeflow community <strong>does not endorse or certify</strong> any specific distribution.
 
 In the near future, there are plans to introduce <a href="https://github.com/kubeflow/community/blob/master/proposals/kubeflow-conformance-program-proposal.md">conformance testing for distributions</a>, you may track progress on this initiative by following <a href="https://github.com/kubeflow/kubeflow/issues/6485">kubeflow/kubeflow#6485</a>.
 {{% /alert %}}
 
-### Active Distributions
+Some distributions can be deployed on all certified Kubernetes
+distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>, some of them
+can be deployed only in specific Kubernetes environment (e.g. EKS or GKE).
 
-The following table lists <b>active distributions</b> that have <b>had a recent release</b> (within the last 6 months).
+The following table lists distributions which are <em>maintained</em> by their respective maintainers.
 
-<div class="table-responsive">
+<div class="table-responsive distributions-table-active">
   <table class="table table-bordered">
-    <thead class="thead-light">
+    <thead>
       <tr>
-        <th>Name</th>
-        <th>Maintainer</th>
         <th>Target Platform</th>
-        <th>Link</th>
         <th>Kubeflow Version</th>
+        <th>Maintainer / Distribution Name</th>
+        <th>Link</th>
       </tr>
     </thead>
     <tbody>
       <tr>
-        <td>Kubeflow on AWS</td>
-        <td>Amazon Web Services</td>
         <td>
-          Amazon Elastic Kubernetes Service (EKS)
+          All Certified Kubernetes Distributions
         </td>
         <td>
-          <a href="https://awslabs.github.io/kubeflow-manifests">Website</a>
+          {{% canonical/latest-version %}}
```

Comment:
We should add the link to Charmed Kubeflow's release notes, like the other distributions:

```suggestion
          {{% canonical/latest-version %}} <sup>[<a href="https://charmed-kubeflow.io/docs/release-notes">Release Notes</a>]</sup>
```